<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>200</width>
      <height>160</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actNewDoc"/>
   <addaction name="actOpenDoc"/>
   <addaction name="actCloseAll"/>
   <addaction name="separator"/>
   <addaction name="actCut"/>
   <addaction name="actCopy"/>
   <addaction name="actPaste"/>
   <addaction name="separator"/>
   <addaction name="actFont"/>
   <addaction name="separator"/>
   <addaction name="actMdi"/>
   <addaction name="actCascade"/>
   <addaction name="actTile"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actNewDoc">
   <property name="text">
    <string>新建文档</string>
   </property>
  </action>
  <action name="actOpenDoc">
   <property name="text">
    <string>打开文档</string>
   </property>
  </action>
  <action name="actCloseAll">
   <property name="text">
    <string>关闭全部</string>
   </property>
  </action>
  <action name="actCut">
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actPaste">
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actFont">
   <property name="text">
    <string>字体设置</string>
   </property>
  </action>
  <action name="actMdi">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MDI模式</string>
   </property>
  </action>
  <action name="actCascade">
   <property name="text">
    <string>级联展开</string>
   </property>
  </action>
  <action name="actTile">
   <property name="text">
    <string>平铺展开</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
